tests/src/core/importType.js

```diff
@@ -227,8 +227,13 @@ describe('importType(name)', function () {
 
   it('`isExternalModule` works with windows directory separator', function() {
     expect(isExternalModule('foo', {}, 'E:\\path\\to\\node_modules\\foo')).to.equal(true)
+    expect(isExternalModule('foo', {}, '.\\foo')).to.equal(false)
+
     expect(isExternalModule('foo', {
       'import/external-module-folders': ['E:\\path\\to\\node_modules'],
     }, 'E:\\path\\to\\node_modules\\foo')).to.equal(true)
+    expect(isExternalModule('foo', {
+      'import/external-module-folders': ['E:\\path\\to\\node_modules'],
+    }, '.\\foo')).to.equal(false)
```

Comment:
i'm hoping for "false before, true after", not "same before and after". Can we come up with an example where the external-module-folders setting *changes* the result?

Reply:
I'm not sure I understand this.

Setting `import/external-module-folders` did not change the result.

```js
    expect(isExternalModule('foo', {
      'import/external-module-folders': ['E:\\path\\to\\node_modules'],
    }, 'E:\\path\\to\\node_modules\\foo')).to.equal(true)
```

```js
    expect(isExternalModule('foo', {
    }, 'E:\\path\\to\\node_modules\\foo')).to.equal(true)
```

There are all true, and should be true, but before I add `subpath.replace(/\\/g, '/')`, this is false on windows, because it's bug.  How do I test that, if you want reproduction, you just remove this fix, and

```js
    expect(isExternalModule('foo', {
      'import/external-module-folders': ['E:\\path\\to\\node_modules'],
    }, 'E:\\path\\to\\node_modules\\foo')).to.equal(true)
```

this will fail.

Reply:
sure - i appreciate that the bug is that you get `true, false` instead of `true, true`.

However, to prevent different regressions later, i'm hoping to have a test that correctly ensures `false, true`.

Reply:
I went ahead and added something like that.